hw/top_earlgrey/README.md

```diff
@@ -1,72 +1,20 @@
-# Top Earlgrey
+# Earl Grey
 
-## Specification
+Earl Grey is the first OpenTitan 'top', and is designed as a discrete System-on-Chip (SoC).
+It functions as a low-power secure microcontroller that is designed for several use cases requiring hardware security.
```

Comment:
> secure microcontroller

Be careful of making claims about it being outright secure

Reply:
We already use this language on the 'datasheet' page. And in this case, 'Secure Microcontroller' is more an industry-standard term than a security claim.
We should probably create some guidelines for communication to make this area less grey.

Reply:
Ah okay, I wasn't aware it's an industry term

> We should probably create some guidelines for communication to make this area less grey.

That would be nice

In general I would be in favour of our docs using fewer industry terms in general (e.g. IP, VIP, etc etc) but that's probably a hard thing to push for

Reply:
I agree, "secure microcontroller" or "secure MCU" is a term used by other vendors to name a specific MCU cathegory.
I think it's ok to keep.

@vogelpi / @moidx WDYT?